starwars_api/models.py

```diff
@@ -28,7 +34,10 @@ def all(cls):
         later in charge of performing requests to SWAPI for each of the
         pages while looping.
         """
-        pass
+        if cls.RESOURCE_NAME == 'people':
```

Comment:
An alternative to the explicit checking  here would be to have some sort of dict keyed to `RESOURCE_NAME` and mapping to which, well, resource  needed. (We went that route, though in retrospect, the proper general way would probably be something like using decorators when defining the relevant classes that'd automatically register them into the appropriate dicts, or something like that, rather than having multiple places stuff has to be manually edited.